app/scripts/templates/reset_password.mustache

```diff
@@ -28,14 +28,14 @@
         {{/forceEmail}}
       </div>
 
-      <div class="button-row button-row-warning">
-        <button id="submit-btn" type="submit">{{#t}}Begin reset{{/t}}</button>
+      <div class="button-row">
+        <button id="submit-btn" type="submit" class="warning-button">{{#t}}Begin reset{{/t}}</button>
       </div>
     </form>
 
     {{#canGoBack}}
     <div class="links">
-      <button class="remember-password button-link">{{#t}}Remember password? Sign in{{/t}}</button>
+      <a href="#" class="remember-password">{{#t}}Remember password? Sign in{{/t}}</a>
```

Comment:
@vladikoff - you mentioned this could not be a link for OAuth reasons, can you explain more? This doesn't link to an external site and seems like it should be OK to use a link like we do elsewhere.

Reply:
ah it should be fine now, it used to go back in `history` and i think that was causing problems. seems okay in my local testing at this time